.travis.yml

```diff
@@ -7,9 +7,10 @@ cache:
         - $HOME/.composer/cache/files
 
 php:
-    - 7.1
     - 7.2
```

Comment:
```suggestion
```

Reply:
The package actually supports PHP 7.1 I chose to remove PHP 7.1 because the used PHPUnit version does not support it, but the package is actually compatible.

This change is up to @Nyholm I guess.

Reply:
As PHP 7.1 is EOL, I guess we can safely stop supporting it. :)